dist/css/bootstrap-slider.css

```diff
@@ -1,308 +0,0 @@
-/*! =======================================================
-                      VERSION  9.8.0              
```

Comment:
can you remove any changes to the dist directory from this PR? this are compiled files that get generated when we publish a new version